### ISSUE TITLE
out8-bcu1: detection of bus voltage fail not implemented

### ISSUE THREAD

**Issue by Darthyson** [selfbus/software-arm-incubation#21]
see also [https://selfbus.org/forum/viewtopic.php?p=4716#p4716](https://selfbus.org/forum/viewtopic.php?p=4716#p4716)

**Comment by dallmair** [comment]
It seems to me there is at least code in out8-bcu1 version 5.10 regarding this -- has this been implemented in the meantime?

**Comment by Darthyson** [comment]
Thats correct. Bus voltage failure/return handling was implemented with [PR#37](https://github.com/selfbus/software-arm-incubation/pull/37). IHMO it still needs more field testing. I myself have been running an out8-bcu1 with bus voltage failure/return handling since then without any problems.
